pkgs/python-lsp-server/default.nix

```diff
@@ -40,19 +40,20 @@ pypkgs.buildPythonPackage rec {
     python-lsp-jsonrpc
     setuptools # `pkg_resources`imported in pylsp/config/config.py
     ujson
-    autopep8
-    flake8
-    mccabe
-    pycodestyle
-    pydocstyle
-    pyflakes
-    pylint
-    rope
     toml
     whatthepatch
+
+    # extras
     yapf
     pyflakes
     rope
+    # autopep8
+    # flake8
+    # mccabe
+    # pycodestyle
+    # pydocstyle
+    # pyflakes
+    # pylint
```

Comment:
Are these commented out on purpose?

Reply:
I thought I leave these here so in the future we know what the available ones are.